README.md

```diff
@@ -52,6 +52,21 @@ Default value: `['package.json']`
 
 Maybe you wanna bump 'component.json' instead? Or maybe both: `['package.json', 'component.json']`? Can be either a list of files to bump (an array of files) or a grunt glob (e.g., `['*.json']`).
 
+Alternatively, you can use different regexes to your target files:
+
+```js
+files: [
+  {
+    path: 'index.html',
+    regexp: /(dist\/\w+\/\w+[-])(([\d-]+\.)+[\d]+)(([.\w]+['"]))/g
+  },
+  {
+    path: ['package.json', 'bower.json']
+  }
+]
+```
+The example above replaces urls like `dist/js/app-1.0.1.js` or `dist/css/style-1.0.1.css`
```

Comment:
this is not a good example. Grunt-bump shouldn't be used for this since its not renaming those files.

`This can be useful for replacing versions in non-JSON files.` Although that is part of bigger change I want to make where you specify the key in a JSON file to replace the version with and then the regex is only for nonJSON files